composer.json

```diff
@@ -44,7 +44,7 @@
         "openeuropa/behat-transformation-context": "~0.1",
```

Comment:
Current render is:

<img width="833" alt="Screenshot 2020-11-23 at 12 05 44" src="https://user-images.githubusercontent.com/1183951/99945029-f25c4b80-2d84-11eb-9e3d-00bdabb709a8.png">

but it doesn't looks like in the spec.

Also isn't clear should we use plural form "Identifications" if Identifier code field have multiple values.

Reply:
I managed to get this but still not perfectly aligned:
<img width="901" alt="description + files" src="https://user-images.githubusercontent.com/22908988/100110096-3e86b900-2e75-11eb-94f5-c6d52babb4f5.png">

Description tpl:
```<div class="ecl-row">
  {% if body is not empty %}
    <div class="ecl-col-12 ecl-col-lg-9">{{ body }}</div>
  {% endif %}
  {% if image is not empty %}
    <div class="ecl-col-12 ecl-col-lg-3">{{ image }}</div>
  {% endif %}
</div>


Documents tpl:
<div class="ecl-row">
  <div class="ecl-col-12">
    {%- include 'field--multiple-values.html.twig' -%}
  </div>
</div>
```

Reply:
Or might be we should change Thumbnail image size? 192px width doesn't fit current layout.

Reply:
I'd make sure that that image is responsive, so it adapts to its container. But let's create a follow up for this.

Reply:
Related ticket https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EWPP-563 has been created for it.

Reply:
Media container solved issue:
<img width="977" alt="Screenshot 2020-11-26 at 19 48 24" src="https://user-images.githubusercontent.com/1183951/100376433-8eee4a00-3020-11eb-8f0d-3957d69d1107.png">